packages/core/src/main.ts

```diff
@@ -176,16 +176,20 @@ export async function createLibp2pInstance(
         enabled: false
       },
       ping: {
-        protocolPrefix
+        // libp2p automatically adds a leading `/`
+        protocolPrefix: protocolPrefix.slice(1)
```

Comment:
This seems unsafe if in the future somebody removes that prefix beforehand already. Better check that there is a leading slash before removing it.

Reply:
changed, now checking properly